### PR TITLE
CFD-292 Homepage : Filter content (Logged-in users)

### PR DESCRIPTION
**Pull request by HelenaEksler**
#482

### CODE REVIEW

**Comment by HelenaEksler** [comment]
selection_211
selection_212

**Comment by HelenaEksler** [comment]
@nader77  PR is ready for the code review.

File: capacity4more/modules/c4m/features/c4m_features_homepage/plugins/content_types/filter/filter.inc [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  return $block;
+}
+
+/**
+ * Create form element for homepage filter.
+ *
+ * @return mixed
+ */
+function c4m_features_homepage_filter_form_create() {
+  $options = og_get_entity_groups() ? array('groups' => t('My groups')) : array();
```
Comment (nader77) [inline capacity4more/modules/c4m/features/c4m_features_homepage/plugins/content_types/filter/filter.inc]: Shouldn't ``og_get_entity_groups()`` be ``og_get_groups_by_user()`` ?

**Comment by nader77** [comment]
@HelenaEksler Does this change anything in the content or just a form ?

**Comment by HelenaEksler** [comment]
@nader77 Fixed

**Comment by nader77** [comment]
@HelenaEksler Looks good, Assign to Peter when green.

**Pull request merged**